README.md

```diff
@@ -52,6 +52,11 @@ optional arguments:
                         Output format
   -o OUTPUT, --output OUTPUT
                         Output filename
+  -s {check1} [check2...] --skip-checks {check1} [check2...]
+                        Skips the any check functions contained within the
+                        check suites selected which have the same function
+                        name(s) as the string(s) specified.  May be specified
+                        multiple times in order to skip multiple checks.
```

Comment:
It was updated; specifically I added "May be specified multiple times in order to skip multiple checks."